(Feature) Native app Zoom integration for Android

serdiukov-o-nordwhale opened the pull request:
# Description

Enabled FV for Android

Bugfixes: 
- profile initialisation on low-end smartphones
- crashed style props in FV components

About #1689 #1897 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

serdiukov-o-nordwhale commented:
@omerzam The changes in the styles you did in the scope of #2913 caused native app to crash.
I fixed some styles to avoid it but i have issues with styling face example on the Instructions version "B"

Could you please look on this and fix ?

serdiukov-o-nordwhale commented on src/lib/updates/profileFix.js:
```diff
@@ -31,7 +31,7 @@ const checkProfile = async (lastUpdate, prevVersion, log) => {
           // set retry & timeout for each field separately
           flushField(field),
-          timeout(1000, `fixProfile: '${field}' field flush timeout`),
+          timeout(2500, `fixProfile: '${field}' field flush timeout`),
```
on Nokia 2.4 (an i suppose on the other low-end smartphones) Gun runs slowly. 1sec isn't enough for .decrypt operation. So i increased timeout because i always got fixProfile exception

sirpy commented:
> @omerzam The changes in the styles you did in the scope of #2913 caused native app to crash.
> I fixed some styles to avoid it but i have issues with styling face example on the Instructions version "B"
> 
> Could you please look on this and fix ?
> Also please check, am i fixed other styles correctly.

do you know which styles/element are crashing?

sirpy commented on src/lib/updates/profileFix.js:
```diff
@@ -2,7 +2,7 @@ import userStorage from '../gundb/UserStorage'
 import { retry, timeout } from '../utils/async'
 
-const fromDate = new Date('2021/01/20')
+const fromDate = new Date('2021/02/10')
```
not required. this will cause everyone that ran it, to run it again

omerzam commented:
> @omerzam The changes in the styles you did in the scope of #2913 caused native app to crash.
> I fixed some styles to avoid it but i have issues with styling face example on the Instructions version "B"
> 
> Could you please look on this and fix ?
> Also please check, am i fixed other styles correctly.

Maybe we should avoid displaying "B" on native altogether to avoid this issue, AB test is not a must on native anyway.
we can do that by setting the REACT_APP_AB_TEST_PERCENTAGE to 0 or 1 (depends on what set of screens you want to show

other than that do you know what exactly caused the issue?

serdiukov-o-nordwhale commented:
> > @omerzam The changes in the styles you did in the scope of #2913 caused native app to crash.
> > I fixed some styles to avoid it but i have issues with styling face example on the Instructions version "B"
> > Could you please look on this and fix ?
> > Also please check, am i fixed other styles correctly.
> 
> do you know which styles/element are crashing?

@omerzam @sirpy 
`display: contents` and `position: static`
i added Platform.select in such places

sirpy commented:
@serdiukov-o-nordwhale fix tests and merge
